desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py

```diff
@@ -441,11 +441,6 @@ def close_session(self, session):
 
   @query_error_handler
   def autocomplete(self, snippet, database=None, table=None, column=None, nested=None, operation=None):
-    if snippet['type'] == 'phoenix':
-      if database:
-        database = database.upper()
-      if table:
-        table = table.upper()
```

Comment:
Why was this `upper()` explicitly done for `phoenix`?

@agl29 any particular reasons?

Reply:
IIRC there are some reasons why we were using only uppercase.
one of the reason was that unquoted identifiers are converted to uppercase, while identifiers between double quotes are case sensitive. So we converted all the things into uppercase to overcome this issue (some discussion happened with phoenix team).
We have internal Jira to track this issue -> https://jira.cloudera.com/browse/CDPD-21683
And I think this PR will break the right assist. We need to check how we can improve the case sensitivity.

Reply:
I just did a test case and it works fine... and have posted the same information in the mentioned cdpd jira.

Reply:
Created CDPD-61756 to track the right assist.

Reply:
@agl29 @quadoss - Please don't mention JIRA IDs and JIRA links in upstream code reviews and PRs.